### PR TITLE
Docs: Add new release schedule and structure

### PR DESCRIPTION
**Pull request by skmp**
See #1381 for the initial proposal

### CODE REVIEW

**Comment by skmp** [comment]
Looks good to everyone?

File: docs/Release schedule and structure.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Feature "Freeze" Windows
+---
+We do a soft "feature freeze" on the week before release, to allow for the beta builds to be tested.
+During this window, experimental/untested changes should not be merged.
+
+Release Testing / QA
+---
+We depend on the public beta and people complaining right now. 
+If you're interested to do QA testing around releases, please let us know in #1225, and/or join our [discord](http://chat.reicast.com)
```
[the remark this answers is not in the document]
Comment (AbandonedCart) [reply docs/Release schedule and structure.md]: https://github.com/reicast/reicast-emulator/issues/1384#issuecomment-422892770

[the remark this answers is not in the document]
Comment (baka0815) [reply docs/Release schedule and structure.md]: Yep, saw that. Same opinion here.

[the remark this answers is not in the document]
Comment (skmp) [reply docs/Release schedule and structure.md]: Well, I don't visit the forums as much personally. It is a fair point though. Merging this, and we can continue the discussion ni #1384 as @AbandonedCart suggested

**Pull request merged**